### ISSUE TITLE
Found an error in the example

### ISSUE THREAD

**Issue by azwc** [KnutZuidema/golio#42]
```
challengers, _ := client.Riot.League.GetChallenger(api.QueueRankedSolo)
```
```
QueueRankedSolo not declared by package api
```

The real path is:
**github.com/KnutZuidema/golio/riot/lol**

**Comment by KnutZuidema** [comment]
Good catch! I refactored the package structure a while ago and must have forgot to adjust the readme.

**Comment by KnutZuidema** [comment]
resolved by #58
